mailpile/commands.py

```diff
@@ -2172,7 +2172,7 @@ def command(self):
                                    'access_denied': access_denied
                                })
         else:
-            return self._success(_('Displayed settings'), result=result)
+            return self._success(_('Display settings'), result=result)
 
```

Comment:
Hmm.. the message is telling the user what happened, thus the past tense. If that is wrong, we have a lot more messages that need changing - for consistency this should stay unchanged IMO.